src/ValueParsers/IsoTimestampParser.php

```diff
@@ -187,16 +187,18 @@ private function getPrecisionFromYear( $year ) {
 	 * @return string URI
 	 */
 	private function getCalendarModel( array $timeParts ) {
-		$calendarModelName = $timeParts[7];
+		list( $sign, $unsignedYear, , , , , , $calendarModel ) = $timeParts;
```

Comment:
That's how the style must be in my opinion: always a space after a comma. The fact that some elements are skipped doesn't make the whitespace rule obsolete.